Testsuite failures on Fedora 43+

The tests succeed on F42, with those packages:

```
graphviz-libs-12.2.1-8.fc42.x86_64
graphviz-12.2.1-8.fc42.x86_64
python3-graphviz-0.20.1-11.fc42.noarch
python3-pygraphviz-1.14-4.fc42.x86_64
```

But fail on F43 with Graphviz 13:

```
graphviz-libs-13.1.1-3.fc43.x86_64
graphviz-13.1.1-3.fc43.x86_64
python3-graphviz-0.20.1-15.fc43.noarch
python3-pygraphviz-1.14-9.fc43.x86_64
```

Failure:
```
+ /usr/bin/pytest -m 'not external_db'
============================= test session starts ==============================
platform linux -- Python 3.14.0rc3, pytest-8.3.5, pluggy-1.6.0
rootdir: /builddir/build/BUILD/eralchemy-1.6.0-build/eralchemy-1.6.0
configfile: pyproject.toml
plugins: cov-5.0.0
collected 63 items / 6 deselected / 57 selected

tests/test_argparse.py ..                                                [  3%]
tests/test_intermediary_to_dot.py FFFF...                                [ 15%]
tests/test_intermediary_to_er.py ....                                    [ 22%]
tests/test_main.py ....................                                  [ 57%]
tests/test_mermaid.py .                                                  [ 59%]
tests/test_parser.py ..............                                      [ 84%]
tests/test_sqla_multi_key.py ....                                        [ 91%]
tests/test_sqla_to_intermediary.py .....                                 [100%]

=================================== FAILURES ===================================
_______________________________ test_all_to_dot ________________________________

    def test_all_to_dot():
        tables = [child, parent]
        relations = [relation]
        output = _intermediary_to_dot(tables, relations)
>       assert_is_dot_format(output)

tests/test_intermediary_to_dot.py:53: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_intermediary_to_dot.py:44: in assert_is_dot_format
    p.start()
/usr/lib64/python3.14/multiprocessing/process.py:121: in start
    self._popen = self._Popen(self)
/usr/lib64/python3.14/multiprocessing/context.py:224: in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/context.py:300: in _Popen
    return Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:35: in __init__
    super().__init__(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_fork.py:20: in __init__
    self._launch(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:47: in _launch
    reduction.dump(process_obj, buf)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

obj = <Process name='Process-1' parent=312 initial>
file = <_io.BytesIO object at 0x7f2a2d9630b0>, protocol = None

    def dump(obj, file, protocol=None):
        '''Replacement for pickle.dump() using ForkingPickler.'''
>       ForkingPickler(file, protocol).dump(obj)
E       _pickle.PicklingError: Can't pickle local object <function assert_is_dot_format.<locals>.run_graph at 0x7f2a2d9b5010>
E       when serializing dict item '_target'
E       when serializing multiprocessing.context.Process state
E       when serializing multiprocessing.context.Process object

/usr/lib64/python3.14/multiprocessing/reduction.py:60: PicklingError
____________________ test_all_to_dot_with_title[Test Title] ____________________

title = 'Test Title'

    @pytest.mark.parametrize(
        "title",
        ("Test Title", "häßlicher Titel!", "<div> -not.parsed_ </div>"),  # codespell:ignore
    )
    def test_all_to_dot_with_title(title):
        tables = [child, parent]
        relations = [relation]
        output = _intermediary_to_dot(tables, relations, title=title)
>       assert_is_dot_format(output)

tests/test_intermediary_to_dot.py:66: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_intermediary_to_dot.py:44: in assert_is_dot_format
    p.start()
/usr/lib64/python3.14/multiprocessing/process.py:121: in start
    self._popen = self._Popen(self)
/usr/lib64/python3.14/multiprocessing/context.py:224: in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/context.py:300: in _Popen
    return Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:35: in __init__
    super().__init__(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_fork.py:20: in __init__
    self._launch(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:47: in _launch
    reduction.dump(process_obj, buf)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

obj = <Process name='Process-2' parent=312 initial>
file = <_io.BytesIO object at 0x7f2a2d81b150>, protocol = None

    def dump(obj, file, protocol=None):
        '''Replacement for pickle.dump() using ForkingPickler.'''
>       ForkingPickler(file, protocol).dump(obj)
E       _pickle.PicklingError: Can't pickle local object <function assert_is_dot_format.<locals>.run_graph at 0x7f2a2d9ebb60>
E       when serializing dict item '_target'
E       when serializing multiprocessing.context.Process state
E       when serializing multiprocessing.context.Process object

/usr/lib64/python3.14/multiprocessing/reduction.py:60: PicklingError
______________ test_all_to_dot_with_title[h\xe4\xdflicher Titel!] ______________

title = 'häßlicher Titel!'

    @pytest.mark.parametrize(
        "title",
        ("Test Title", "häßlicher Titel!", "<div> -not.parsed_ </div>"),  # codespell:ignore
    )
    def test_all_to_dot_with_title(title):
        tables = [child, parent]
        relations = [relation]
        output = _intermediary_to_dot(tables, relations, title=title)
>       assert_is_dot_format(output)

tests/test_intermediary_to_dot.py:66: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_intermediary_to_dot.py:44: in assert_is_dot_format
    p.start()
/usr/lib64/python3.14/multiprocessing/process.py:121: in start
    self._popen = self._Popen(self)
/usr/lib64/python3.14/multiprocessing/context.py:224: in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/context.py:300: in _Popen
    return Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:35: in __init__
    super().__init__(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_fork.py:20: in __init__
    self._launch(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:47: in _launch
    reduction.dump(process_obj, buf)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

obj = <Process name='Process-3' parent=312 initial>
file = <_io.BytesIO object at 0x7f2a2d81b420>, protocol = None

    def dump(obj, file, protocol=None):
        '''Replacement for pickle.dump() using ForkingPickler.'''
>       ForkingPickler(file, protocol).dump(obj)
E       _pickle.PicklingError: Can't pickle local object <function assert_is_dot_format.<locals>.run_graph at 0x7f2a2d9eb690>
E       when serializing dict item '_target'
E       when serializing multiprocessing.context.Process state
E       when serializing multiprocessing.context.Process object

/usr/lib64/python3.14/multiprocessing/reduction.py:60: PicklingError
____________ test_all_to_dot_with_title[<div> -not.parsed_ </div>] _____________

title = '<div> -not.parsed_ </div>'

    @pytest.mark.parametrize(
        "title",
        ("Test Title", "häßlicher Titel!", "<div> -not.parsed_ </div>"),  # codespell:ignore
    )
    def test_all_to_dot_with_title(title):
        tables = [child, parent]
        relations = [relation]
        output = _intermediary_to_dot(tables, relations, title=title)
>       assert_is_dot_format(output)

tests/test_intermediary_to_dot.py:66: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_intermediary_to_dot.py:44: in assert_is_dot_format
    p.start()
/usr/lib64/python3.14/multiprocessing/process.py:121: in start
    self._popen = self._Popen(self)
/usr/lib64/python3.14/multiprocessing/context.py:224: in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/context.py:300: in _Popen
    return Popen(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:35: in __init__
    super().__init__(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_fork.py:20: in __init__
    self._launch(process_obj)
/usr/lib64/python3.14/multiprocessing/popen_forkserver.py:47: in _launch
    reduction.dump(process_obj, buf)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

obj = <Process name='Process-4' parent=312 initial>
file = <_io.BytesIO object at 0x7f2a2d8791c0>, protocol = None

    def dump(obj, file, protocol=None):
        '''Replacement for pickle.dump() using ForkingPickler.'''
>       ForkingPickler(file, protocol).dump(obj)
E       _pickle.PicklingError: Can't pickle local object <function assert_is_dot_format.<locals>.run_graph at 0x7f2a2d9eb320>
E       when serializing dict item '_target'
E       when serializing multiprocessing.context.Process state
E       when serializing multiprocessing.context.Process object

/usr/lib64/python3.14/multiprocessing/reduction.py:60: PicklingError
=========================== short test summary info ============================
FAILED tests/test_intermediary_to_dot.py::test_all_to_dot - _pickle.PicklingE...
FAILED tests/test_intermediary_to_dot.py::test_all_to_dot_with_title[Test Title]
FAILED tests/test_intermediary_to_dot.py::test_all_to_dot_with_title[h\xe4\xdflicher Titel!]
FAILED tests/test_intermediary_to_dot.py::test_all_to_dot_with_title[<div> -not.parsed_ </div>]
================== 4 failed, 53 passed, 6 deselected in 0.21s ==================

--- a/tests/test_intermediary_to_dot.py
+++ b/tests/test_intermediary_to_dot.py
@@ -26,21 +26,21 @@ column_inside = re.compile(
 )
 
 
+def _run_graph(dot):
+    """Runs graphviz to see if the syntax is good."""
+    graph = AGraph()
+    graph = graph.from_string(dot)
+    extension = "png"
+    graph.draw(path="output.png", prog="dot", format=extension)
+    sys.exit(0)
+
+
 # This test needs fixing with move to graphviz
 def assert_is_dot_format(dot):
     """Checks that the dot is usable by graphviz."""
-
     # We launch a process calling graphviz to render the dot. If the exit code is not 0 we assume that the syntax
     # wasn't good
-    def run_graph(dot):
-        """Runs graphviz to see if the syntax is good."""
-        graph = AGraph()
-        graph = graph.from_string(dot)
-        extension = "png"
-        graph.draw(path="output.png", prog="dot", format=extension)
-        sys.exit(0)
-
-    p = Process(target=run_graph, args=(dot,))
+    p = Process(target=_run_graph, args=(dot,))
     p.start()
     p.join()
     assert p.exitcode == 0

